spec/workspace-view-spec.coffee

```diff
@@ -295,39 +295,3 @@ describe "WorkspaceView", ->
 
       modalContainer = workspaceElement.querySelector('atom-panel-container.modal')
       expect(modalContainer.parentNode).toBe workspaceElement
-
-  describe "::saveActivePaneItem()", ->
-    describe "when there is an error", ->
-      it "emits a warning notification when the file cannot be saved", ->
-        spyOn(Pane::, 'saveActiveItem').andCallFake ->
```

Comment:
Super minor, but I wonder if the `Pane` require can be removed from this file now.